bitnami/minio/templates/_helpers.tpl

```diff
@@ -214,7 +214,7 @@ minio: total drives
 Validate values of MinIO(R) - TLS secret must provided if TLS is enabled
 */}}
 {{- define "minio.validateValues.tls" -}}
-{{- if and .Values.tls.enabled (not .Values.tls.secretName) }}
+{{- if and .Values.tls.enabled (not .Values.tls.secretName) (not .Values.tls.existingSecret) (not .Values.tls.autoGenerated) }}
```

Comment:
You should remove this condition (`(not .Values.tls.secretName)`), right? Or are you keeping it to have backwards compatibility?

Reply:
Responding myself: yes, backwards compatibility

Reply:
Yes, `Values.tls.secretName` need to be preserved for backward compatibility.